sdk/src/exporter/otprotocol/recordable.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "src/trace/recordable.h"
```

Comment:
I think headers needed to implement exporters should be placed in `sdk/include`. However, this can be done in a different PR.

Reply:
Moved Recodable into the headers folder.